### DISCUSSION TITLE
back-relations using getOne()

### DISCUSSION THREAD

**Discussion by oscar-costa** [pocketbase/pocketbase#4856] [Q&A] [answered]
Hi
Is it possible to have back-relations using the method getOne()

I was able to access back-relations using getList(), but no success with getOne()

**Code used for getList( )**
```
const records = await pb.collection("posts").getList(1, 30, {
    expand: "comments_via_post.user",
})
```

**Code used for getOne( )**
```
const record1 = await pb.collection('posts').getOne('RECORD_ID', {
    expand: "comments_via_post.user",
});
```

**Comment by ganigeorgiev** [comment, marked as answer]
Both should work.

I've just tested it locally and it works correctly for me. Here is also an example from the live demo (it resets every hour and the API rule had to be adjusted to allow guest access):
https://pocketbase.io/api/collections/users/records/qEooBTHoAGkWUKc?expand=messages_via_author

Double check that your client request satisfy the View api rule of the expanded collections (in the above example "comments" and the collection of the "user" field relation).

If you are still not able to resolve your issue, please export your collections configuration as json (_Admin UI > Settings > Export collections_) and provide an exact code sample how you are sending the request.

**Reply by oscar-costa** [reply]
Thanks ganigeorgiev, you are correct, both codes work.
